src/main/scala/cromwell/binding/Workflow.scala

```diff
@@ -13,11 +15,12 @@ object Workflow {
     }
 
     callNames.groupBy {_.sourceString()}.foreach {
```

Comment:
also while there (I realize this isn't part of the diff) either make it `.foreach(...)` or ` foreach { ...`